tests/components/hassio/test_init.py

```diff
@@ -547,14 +554,14 @@ async def test_service_calls(
     )
     await hass.async_block_till_done()
 
-    assert aioclient_mock.call_count == 22
+    assert aioclient_mock.call_count + len(supervisor_client.mock_calls) == 24
```

Comment:
For reviewers - the increase in call count here is not actually a change. If you look in #125926 you'll see that I decreased each of these numbers by 2 to reflect the fact that mocking had changed for 2 calls we were previously making. Those 2 calls were going through the mocked client library as opposed to the mocked websession.

I realized in this PR that a better way to change these tests was to add the mock calls to these two things together to show that the call count truly had not changed. So as a result I had to increase these numbers back to what they were before the prior PR.